hackathon_site/dashboard/frontend/src/components/dashboard/ItemTable/ItemTable.js

```diff
@@ -294,3 +296,103 @@ export const PendingTable = ({ items, status }) => {
         </Container>
     );
 };
+
+export const UnconnectedBrokenTable = ({
+    items,
+    isVisible,
+    toggleVisibility,
+    onClickViewReport,
+    status,
+}) => {
+    return !items.length ? null : (
+        <Container
+            className={styles.tableContainer}
+            maxWidth={false}
+            disableGutters={true}
+        >
+            <div className={styles.title}>
+                <Typography variant="h2" className={styles.titleBroken}>
```

Comment:
You can directly put `color="secondary"` on line 314 and remove the extra class you made